README.md

```diff
@@ -335,6 +335,50 @@ export const getLoadContext: GetLoadContext = ({ context }) => {
 }
 ```
 
+## AsyncLocalStorage
+
+You can use AsyncLocalStorage, which is supported by Node.js, Cloudflare Workers, etc.
+You can easily store context using Hono's Context Storage Middleware.
+
+```ts
+// server/index.ts
+import { Hono } from 'hono'
+import { contextStorage } from 'hono/context-storage'
+
+export interface Env {
+  Variables: {
+    headers: Headers
+    key: string
+    // db: DatabaseConnection // It's also a good idea to store database connections, etc.
+  }
+}
+
+const app = new Hono<Env>()
+
+app.use(contextStorage())
+
+app.use(async (c, next) => {
+  c.set('headers', c.req.raw.headers)
+  c.set('message', 'Hello!')
+
+  await next()
+})
+
+export default app
+```
+
+You can retrieve and process the context saved in Hono from Remix as follows:
+
+```ts
+// app/routes/_index.tsx
+import type { Env } from 'server'
+import { getContext } from 'hono/context-storage' // It can be called anywhere for server-side processing.
+
+export const loader = () => {
+  const cookie = getContext<Env>().headers.get('Cookie')
```

Comment:
I'm just curious. Is it difficult to get a `Header` object in `loader` without this way using `AsyncLocalStorage`?

Reply:
It's very simple.

```ts
export const loader = ({ request }: LoaderFunctionArg) => {
  const cookie = request.headers.get('Cookie')
}
```

Remix can be retrieved easily like this. Originally, I wanted to write it in the server processing function called from `loader` rather than in `loader`, but I was worried that doing so would increase the amount of sample program description in the README.

Shall I write it in an easy-to-understand way?